server/config.js

```diff
@@ -2,9 +2,24 @@
 import findUp from 'find-up'
 import uuid from 'uuid'
 
-const cache = new Map()
+type WebpackConfig = *
```

Comment:
Is it possible to import these types from `flow-typed` or from the webpack repo directly?

Reply:
There's no type definition for webpack. This is just temporary to make sure flow doesn't error 👍